stacks/data/generate/data_workloads.py

```diff
@@ -13,6 +13,19 @@
 GENERATE_PACKAGE_NAME = "stacks.data.generate"
 TEMPLATES_DIRECTORY = "templates"
 
+def get_os_path_separator():     
+    """Get the appropriate path separator for the current operating system."""    
+    if os.name == 'posix':         
+        return '/'  
+    # Unix/Linux/macOS    
+    elif os.name == 'nt':         
+        return '\\'  
+    # Windows    
+    else:         
+        return '/'  
+    # Default to '/' for other systems# Example usage:
+path_separator = get_os_path_separator()
```

Comment:
I would move this line within the `generate_pipeline` function where it gets used.